api/explorer.go

```diff
@@ -20,6 +22,8 @@ type (
 		MinerPayoutIDs []types.SiacoinOutputID `json:"minerpayoutids"`
 		Transactions   []ExplorerTransaction   `json:"transactions"`
 		RawBlock       types.Block             `json:"rawblock"`
+		HexBlock       string                  `json:"hexblock"`
+		HexBlockID     string                  `json:"blockid"`
```

Comment:
`HexBlockID` can probably be `BlockID` and have `types.BlockID` as a type.
`types.BlockID` implements `String` and `MarshalJSON` which should implicitly convert it to a hex string in `WriteJSON`.